CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
-## v2.0.0-beta.2 / 2017-08-18
+## v2.0.0-beta.2 / 2017-09-12
```

Comment:
Umm, overwriting beta2 here?

Reply:
Yes, forgot the beta number though.

Reply:
That's how we've done it for all non-stable releases so far.